recipes/nrtk-explorer/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "nrtk-explorer" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nrtk_explorer-{{ version }}.tar.gz
+  sha256: 7e7c915da91a3b5d2124889f435ddf121dd35079c364e8c8571031490e724686
+
+build:
+  entry_points:
+    - nrtk-explorer = nrtk_explorer.app:main
+    - nrtk-explorer-embeddings = nrtk_explorer.app.embeddings:embeddings
+    - nrtk-explorer-tranforms = nrtk_explorer.app.transforms:transforms
+    - nrtk-explorer-filtering = nrtk_explorer.app.filtering:filtering
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - hatchling
+    - nodejs
```

Comment:
This should probably be in build.

Reply:
@xhochy Good catch, ready to review. The windows failing build is expected as explained here https://github.com/conda-forge/staged-recipes/pull/26184#discussion_r1605473533